webui/templates/index.html

```diff
@@ -239,39 +219,53 @@ <h4 class="col-sm-6">Database metrics</h4>
             });
         }
         modules["live"] = function () {
+            $.getJSON("/module/live", function (result) {
+                toggleLED("#led-live", result.normal)
+            });
         };
 
         modules["target"] = function () {
+            $.getJSON("/module/target", function (result) {
+                var on = 1;
+                for(var key in result){
+                    if(result[key].Scanned){
+                        on = on & result[key].Normal
+                    }
+                }
+                toggleLED("#led-target", on);
+            });
         };
 
-        modules["disk"] = function(module){
+        modules["disk"] = function(){
             $.getJSON("/module/disk", function (result) {
                 var on = 1;
                 for(var key in result){
                     on = on & result[key].partition_status.space;

```

Comment:
Using bitwise operators is not really recommended - see http://stackoverflow.com/questions/14871052/why-use-logical-operators-when-bitwise-operators-do-the-same.